ts/Series/TiledWebMap/TiledWebMapSeries.ts

```diff
@@ -0,0 +1,308 @@
+     * @return {Highcharts.PositionObject}
+     *         Array of x and y positions of the tile
+     */
+
+    public lonLatToTile(
+        lonLat: Highcharts.MapLonLatObject,
+        zoom: number
+    ): PositionObject {
+        const { lon, lat } = lonLat,
+            xTile = Math.floor(
+                (lon + 180) / 360 * Math.pow(2, zoom)
+            ),
+            yTile = Math.floor(
+                (
+                    1 - Math.log(
+                        Math.tan(lat * Math.PI / 180) +
+                        1 / Math.cos(lat * Math.PI / 180)
+                    ) / Math.PI
+                ) /
+                2 * Math.pow(2, zoom)
+            );
+        return { x: xTile, y: yTile };
+    }
+
+    /**
+     * Convert tile to map coordinates in longitude/latitude
+     *
+     * @function Highcharts.MapView#tileToLonLat
+     * @since  next
+     * @param  xTile
+     *         Position x of the tile
+     * @param  yTile
+     *         Position y of the tile
+     * @param  zTile
+     *         Zoom of the tile
+     * @return {Highcharts.MapLonLatObject}
+     *         The map coordinates
+     */
+
+    public tileToLonLat(
+        xTile: number,
+        yTile: number,
+        zTile: number
+    ): Highcharts.MapLonLatObject {
+        const lon = xTile / Math.pow(2, zTile) * 360 - 180,
+            n = Math.PI - 2 * Math.PI * yTile / Math.pow(2, zTile),
+            lat = (
+                180 /
+                Math.PI * Math.atan(0.5 * (Math.exp(n) - Math.exp(-n)))
+            );
+        return { lon, lat };
+    }
+
+    public drawPoints(): any {
+        if (!this.tiles) {
+            this.tiles = {};
+        }
+        if (!this.transformGroups) {
+            this.transformGroups = [];
+        }
+        if (!this.chart.mapView) {
+            return;
+        }
+
+        const {
+                chart,
+                tiles,
+                transformGroups
+            } = this,
+            options = this.options,
+            provider = options.provider,
+            mapView: any = chart.mapView,
+            { zoom } = mapView,
+            zoomCeil = Math.ceil(zoom);
+
+        if (mapView.projection) {
+            // Always true for tile maps
+            mapView.projection.hasCoordinates = true;
+        }
+
+        if (!transformGroups[zoomCeil]) {
+            transformGroups[zoomCeil] = chart.renderer.g().add(this.group);
+        }
+        const origin = mapView.lonLatToPixels({
+            lon: -180,
+            lat: 85.0511287798
+        });
+        transformGroups[zoomCeil].attr({
+            translateX: origin.x,
+            translateY: origin.y
+        });
+
+        const replaceVariables = (
+            url: string,
+            x: number,
+            y: number,
+            zoom: number
+        ): string => url
+            .replace('{x}', x.toString())
+            .replace('{y}', y.toString())
+            .replace('{zoom}', zoom.toString());
```

Comment:
I've seen some implementations using `{z}` for zoom, probably we should replace that too.